socorro/unittest/external/es/test_crashstorage.py

```diff
@@ -190,6 +192,84 @@ def test_index_crash_indexable_keys(self):
         processed_crash = doc["_source"]["processed_crash"]
         assert list(sorted(processed_crash.keys())) == ["date_processed", "uuid"]
 
+    def test_index_crash_mapping_keys(self):
+        """Test indexing a crash that has keys not in the mapping
+
+        Indexing a crash that has keys that aren't in the mapping for the index
+        should cause those keys to be removed from the crash.
+
+        """
+        # The test harness creates an index for this week and last week. So let's create
+        # one for 4 weeks ago.
+        now = utc_now()
+        four_weeks_ago = now - timedelta(days=28)
+
+        field = "user_comments"
```

Comment:
I used this field because I'm pretty sure it'll be around for a while. If we ever drop this field, we'll need to update this test.